<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenableEditDlg</class>
 <widget class="QDialog" name="OpenableEditDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>174</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Is locked</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="fIsLocked">
     <item>
      <property name="text">
       <string>False</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>True</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Lock ID</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="fLockId"/>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Lock level</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="fLockLevel"/>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Is closed</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="fIsClosed">
     <item>
      <property name="text">
       <string>False</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>True</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Script ID</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="fScriptID"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OpenableEditDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OpenableEditDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
